### PR TITLE
CORE-6150: Simulator now runs liquibase migration scripts

### PR DESCRIPTION
**Pull request by LizR3**
Also implemented missing page functionality in named and paged queries
Don't forget to close Simulator, fibers or persistence service if any persistence service is created

### CODE REVIEW

File: simulator/runtime/src/test/kotlin/net/corda/simulator/runtime/flows/DefaultServicesInjectorTest.kt [hunk 1]
```diff
@@ -28,5 +28,7 @@ class DefaultServicesInjectorTest {
         assertNotNull(flow.memberLookup)
         assertNotNull(flow.signingService)
         assertNotNull(flow.signatureVerificationService)
+
+        fiber.close()
```
Comment (driessamyn) [inline simulator/runtime/src/test/kotlin/net/corda/simulator/runtime/flows/DefaultServicesInjectorTest.kt]: As SimFiber is closable, the Kotlin idiomatic style (and "safer") would be to use `SimFiberBase().use { fiber -> ....`

**Review by driessamyn — DISMISSED** [review]
Just a nit-picky comment, but good to go.

**Review by driessamyn — APPROVED** [review]
LGTM

**Comment by LizR3** [comment]
Closing as these changes will be merged from the DP2 version of this PR.

**Pull request closed without merging**